hpobench/benchmarks/ml_mmfb/svm_benchmark.py

```diff
@@ -0,0 +1,94 @@
+from typing import Union, Dict
+
+import ConfigSpace as CS
+import numpy as np
+from ConfigSpace.hyperparameters import Hyperparameter
+from sklearn.svm import SVC
+
+from hpobench.dependencies.ml.ml_benchmark_template import MLBenchmark
+
+
+class SVMBaseBenchmark(MLBenchmark):
```

Comment:
For SVM, I'd create three classes again, to keep the parity of definitions
where `SVMBenchmark` and `SVMBenchmarkMF` are the same definitions (since only 1 fidelity)
while `SVMBenchmarkBB` is the black-box case